File: src/main/java/com/matchFit/post/dto/response/GetPostsCalender.java
package com.matchFit.post.dto.response;

import java.util.List;
import lombok.AllArgsConstructor;
import lombok.Getter;

@Getter
@AllArgsConstructor
public class GetPostsCalender {
    private final List<GetPostCalender> posts;

    public static GetPostsCalender from(List<GetPostCalender> postsCalender) {
        return new GetPostsCalender(postsCalender);
    }
}

File: src/main/kotlin/com/matchFit/post/dto/response/GetPostsCalender.kt
package com.matchFit.post.dto.response


data class GetPostsCalender(
    val posts: List<GetPostCalender>
) {
    companion object {
        fun from(postsCalender: List<GetPostCalender>): GetPostsCalender {
            return GetPostsCalender(postsCalender)
        }
    }
}
